prep-html-skeleton/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Skelton Exercise</title>
+
+<body>
+  <h1>I built a proper HTML skeleton!</h1>
+</body>
+
+
+
+
+
+</head>
```

Comment:
Hi Edwin! For your HTML page structure you'll want to start and end the `<head></head>` element before the start of the body. Like

```html
<head>
  <title></title>
</head>
<body>
  <!-- visible stuff -->
</body>
```